vllm/entrypoints/llm.py

```diff
@@ -508,9 +508,14 @@ def apply_model(self, func: Callable[[nn.Module], _R]) -> list[_R]:
         """
         Run a function directly on the model inside each worker,
         returning the result for each of them.
+
+        :::{important}
+        To reduce the overhead of data transfer, avoid returning large arrays
+        or tensors from this method. If you must return them, make sure you
+        move them to CPU first to avoid taking up additional VRAM! 
+        :::
```

Comment:
Docs is failing because of this

```prolog

ERROR   -  mkdocstrings: With sys.path = ['/home/docs/checkouts/readthedocs.org/user_builds/vllm/checkouts/18465', '/home/docs/.asdf/installs/python/3.12.10/lib/python312.zip', '/home/docs/.asdf/installs/python/3.12.10/lib/python3.12', '/home/docs/.asdf/installs/python/3.12.10/lib/python3.12/lib-dynload', '/home/docs/checkouts/readthedocs.org/user_builds/vllm/envs/18465/lib/python3.12/site-packages'], accessing '{important}' raises ModuleNotFoundError: No module named '{important}'
--
1088 | ERROR   -  Error reading page 'api/vllm/index.md':
1089 | ERROR   -  Could not collect '{important}'

```

iirc the myst syntax is

```suggestion
        !!! important
        	To reduce the overhead of data transfer, avoid returning large arrays
        	or tensors from this method. If you must return them, make sure you
        	move them to CPU first to avoid taking up additional VRAM! 
```